Makefile

```diff
@@ -138,25 +149,17 @@ else
 	@echo "No supported deployment method available. Install plugin manually."
 endif
 
-## Runs any lints and unit tests defined for the server and webapp, if they exist.
+# test runs any lints and unit tests defined for the server and webapp, if they exist
 .PHONY: test
 test: server/.depensure webapp/.npminstall
 ifneq ($(HAS_SERVER),)
-	cd server && $(GO) test -race -v ./...
+	cd server && $(GO) test -race -v -coverprofile=coverage.txt ./...
 endif
 ifneq ($(HAS_WEBAPP),)
 	cd webapp && $(NPM) run fix;
 endif
 
-## Creates a coverage report for the server code.
-.PHONY: coverage
-coverage: server/.depensure webapp/.npminstall
```

Comment:
Why did you remove this command?

Reply:
I don't know i have just copy/paste from mattermost-plugin-gitlab which is a copy/paste of mattermost-plugin-github !!? Maybe I need to copy/paste from another plugin !!?? DO you have a base reference for makefile ?

Reply:
Yes. Use https://github.com/mattermost/mattermost-plugin-sample/blob/master/Makefile. This is the "latest version" of the Makefile.

Reply:
Please note that the sample plugin recently moved to go modules.